src/helm/proxy/clients/anthropic_client.py

```diff
@@ -122,18 +132,16 @@ def do_it():
 
                     def do_it_error():
                         # NOTE(josselin): This is HACKY.
-                        # It is used two solve 2 API eerors:
+                        # It is used to solve 2 API errors:
                         # - "Prompt must contain anthropic.AI_PROMPT". Usually caused by some truncations
                         # that removes the suffix.
-                        # - Prompt length + max_tokens exceeds nax (9192). This is because we request too many tokens.
+                        # - Prompt length + max_tokens exceeds max (9192). This is because we request too many tokens.
                         # In practice it is very rare but it will fail the runs so we add an error message
                         # as the completion instead.
                         return {"completion": "[HELM] Prompt too long. [ERROR] " + error_string}
```

Comment:
Raise an exception here for now, so we don't pollute the cache with incorrect completions.